host_guest/cyclodextrin_derivatives/README.md

```diff
@@ -1,6 +1,6 @@
-# The SAMPL7 Cyclodextrin Derivatives Challenge
+README.md# The SAMPL7 Cyclodextrin Derivatives Challenge
```

Comment:
Looks like you acidentally inserted a "README.md" remark where it doesn't belong?

Reply:
I think I was copying/pasting info from the readme file and accidently inserted "README.md". Strange. Fixed now!